package.json

```diff
@@ -4,12 +4,7 @@
   "private": false,
   "description": "The Open Source Routing Machine is a high performance routing engine written in C++14 designed to run on OpenStreetMap data.",
   "dependencies": {
-    "@mapbox/node-pre-gyp": "^1.0.9",
```

Comment:
`cheap-ruler`, `mkdirp` and `rimraf` are used in tests only
`nan` and `node-cmake` are needed to only build binary and not needed to use package after it

Reply:
Tbh it would be great to somehow check that package will 100% work without them on some clean test project, but not sure how to properly do that.